browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -668,6 +668,13 @@ L.Control.JSDialogBuilder = L.Control.extend({
 				builder._controlHandlers[child.type](right, child, builder);
 		}
 
+		if (data.vertical === 'true' || data.vertical === true) {
```

Comment:
we are done with the conversion of types, so currently we should get always boolean in JSON, no need to handle also string :)